x/ammswap/client/cli/query.go

```diff
@@ -64,6 +67,48 @@ $ okexchaincli query swap pool-info eth-355
 	}
 }
 
+// GetCmdQueryBuyAmount queries amount of base/quote token by the given amount of quote/base token
+func GetCmdQueryBuyAmount(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	return &cobra.Command{
+		Use:   "amount [quote-token-amount] [base-token]",
```

Comment:
baseAmount represents the token on the left side of the product, and quoteAmount represents the token on the right side of the product. Here, amountToBuy and amountToSell may be more appropriate